### PR TITLE
3.1.17 release update release notes and charts and values

### PR DESCRIPTION
**Pull request by wanlonghenry**
3.1.17 release update release notes and charts and values

### CODE REVIEW

**Review by jatakiajanvi12 — DISMISSED** [review]

File: ReleaseNotes.md [hunk 1]
```diff
@@ -8,6 +8,53 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 01/31/2024 -
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.17 (linux)
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.17 (windows)
+##### Release branch: https://github.com/microsoft/Docker-Provider/commits/release-ciprod-3.1.17/
+##### Current dependencies
+- Linux
+  - [CBL-Mariner 2.0.20231004](https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20231004-2.0)
```
Comment (ganga1980) [inline ReleaseNotes.md]: CBL Mariner version is  2.0.20240123  https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20240123-2.0
image

File: ReleaseNotes.md [hunk 2]
```diff
@@ -8,6 +8,53 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
...
+- Windows
+  - Ruby - 3.1.1
+  - Fluent-bit - 2.0.14
+  - Telegraf - 1.24.2
+  - Fluentd - 1.14.6
+  - Windows AMA - 46.9.43
+  - Golang - 1.20.5
+##### Code change log
+## What's Changed
+- Common
```
Comment (ganga1980) [inline ReleaseNotes.md:33]: This commit missing from change log
https://github.com/microsoft/Docker-Provider/pull/1155.

This should be added under "common"

File: ReleaseNotes.md (hunk 1 above)
Comment (wanlonghenry) [reply ReleaseNotes.md]: added

File: ReleaseNotes.md (hunk 2 above)
Comment (wanlonghenry) [reply ReleaseNotes.md:33]: we cut off at [1152](https://github.com/microsoft/Docker-Provider/pull/1152)   do you want to cherry pick this change in this release?

Comment (ganga1980) [reply ReleaseNotes.md:33]: No. sorry, its my bad.

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**